draft-ietf-rtcweb-jsep.xml

```diff
@@ -1140,7 +1140,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTrack" anchor="sec.addTrack">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTrack method adds a MediaStreamTrack to the PeerConnection.
+                    addTrack attempts to minimize the number of transceivers as follows:
+                    The track will be attached to the first compatible transceiver
+                    (of the same media type) which has never had a direction of
+                    "sendonly" or "sendrecv". If no such transceiver exists,
```

Comment:
Transceivers that are stopped are never reused (stopping is permanent), but the m= section slot can be filled with a new transceiver.   I think the text needs to specify:

1.  The media type is the same (already done)
2.  The direction has never been "sendonly" or "sendrecv" (already done)
3.  The transceiver is not stopped (not done yet)

I think we need to add 3 with something like "The track will be attached to the first compatible transceiver (of the same media type and not stopped) ...".